Automattic/vip-go-mu-plugins
# Update/wp parsely 3.0.2

## Description


Update the wp-parsely `3.0` tree to version `3.0.2`:
https://github.com/Parsely/wp-parsely/releases/tag/3.0.2

## Changelog Description


Plugin Updated: wp-parsely 3.0.2

We upgraded the wp-parsely `3.0` plugin tree to version `3.0.2`. This change will only apply if your site has opted to use the latest version of our plugin.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1. Sandbox a test VIP Go site to test against (e.g. https://blog.parse.ly)
1. Confirm current status (what plugin version is loading, are meta data printed, is tracking script loaded, etc.)
1. If plugin version is < `3.0.1`, add the following line to `client-mu-plugins/plugin-loader.php` to force the 3.0 tree to load:
    `add_filter( 'wpvip_parsely_version', function () { return '3.0'; } );` 
1. Confirm current status (what plugin version is loading, are meta data printed, is tracking script loaded, etc.)
    * You should see version `3.0.1` at this point and meta data and tracking script should be loaded as before
1. Copy modified files from this branch to the appropriate location on the sandbox (e.g. the `wp-content/mu-plugins/wp-parsely-3.0` directory)
1. Confirm current status (what plugin version is loading, are meta data printed, is tracking script loaded, etc.)
    * You should see version `3.0.2` at this point and meta data and tracking script should be loaded as before

No new errors or warnings should be apparent.
